lib/pure/os.nim

```diff
@@ -411,6 +417,9 @@ proc relativePath*(path, base: string; sep = DirSep): string {.
     if not f.hasNext(path): break
     ff = f.next(path)
 
+  when not defined(nimOldRelativePathBehavior):
```

Comment:
Please use `since` instead.
Accumulating so many little fallback switches means supporting combinatorial amounts of different behaviours.
I don't think we should go that way..
Also it makes the stdlib harder to extend, since you must always keep all these fallback switches in mind, even when writing your code in `since`.

Reply:
fine with me; @araq shall i go ahead and use `since (1,1): ...` instead of `--useVersion` + `nimOldRelativePathBehavior` ?

Reply:
But this **shouldn't** be backported

Reply:
it's fine the way you wrote it.